integration-tests/narayana-jta/src/main/java/io/quarkus/narayana/jta/TransactionBeanWithEvents.java

```diff
@@ -0,0 +1,134 @@
+package io.quarkus.narayana.jta;
+
+import java.util.concurrent.atomic.*;
+
+import javax.enterprise.context.*;
```

Comment:
More wildcards ;-)
BTW in Idea, you can set single class imports under Settings (Ctrl + Alt + S), Editor -> Code Style -> Java -> Imports
That _should_ give you global settings on this.

Reply:
oh, crap. again... :-/